content/docs/lists-and-keys.md

```diff
@@ -250,12 +250,22 @@ ReactDOM.render(
 
 Keys serve as a hint to React but they don't get passed to your components. If you need the same value in your component, pass it explicitly as a prop with a different name:
 
-```js{3,4}
+```js{12,13}
+function Post(props) {
+  return (
+    <div key={props.id}>
```

Comment:
This `key` attribute isn't necessary and won't actually do anything.

Reply:
Your right. I ment <div id={props.id}>. I think this will make things more clear.